### PR TITLE
Enabling Policheck and TSA Upload in CI builds

### PR DESCRIPTION
**Pull request by Shyam-Gupta**
With move to 1ES templates, Policheck and TSA Upload stopped working as they are not enabled by default. We need to opt in for them to work.
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/winforms/pull/12037)

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -153,17 +159,4 @@ extends:
         enableSigningValidation: false
         enableNugetValidation: false
         enableSourceLinkValidation: false
-        # these param values come from the DotNet-Winforms-SDLValidation-Params azdo variable group
```
Comment (Tanya-Solyanik) [inline azure-pipelines.yml:156]: Why are these removed?

**Review by Tanya-Solyanik — APPROVED** [review]

File: azure-pipelines.yml (hunk 1 above)
Comment (Shyam-Gupta) [reply azure-pipelines.yml:156]: TSA options are now picked up from `tsaconfig.json` file instead of these variables. So they are not required anymore.
